src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/ReadType.java

```diff
@@ -13,7 +13,5 @@ public enum ReadType {
    * Return only document id and transaction id of documents which satisfies the filter conditions
    * as part of response
    */
-  KEY,
-  /** Return only count of documents which satisfies the filter condition */
-  COUNT
+  KEY
```

Comment:
Holding on to this Enum, will be adding SORT_DOCUMENT type in the next PR for in memory sory.